Misc/NEWS.d/next/Core and Builtins/2019-09-28-12-23-23.bpo-38302.hsCNgX.rst

```diff
@@ -0,0 +1 @@
+If :func:`object.__ipow__` returns :const:`NotImplemented` :func:`object.__rpow__` will be called now as expected.
```

Comment:
Just fyi, as @brettcannon posted on the original bug pointing out that this would not call `__pow__` either. I'd recommend changing this to

```rst
If :func:`object.__ipow__` returns :const:`NotImplemented`, the operator will correctly
fall back to :func:`object.__pow__` and :func:`object.__rpow__` now as expected.
```

It might also be worth mentioning this in whatsnew, even though it's obscure it is potentially a change in behavior that people should be informed about.

Reply:
Updated the news entry, and also added a line to What's New